.github/actions/scripts/save-coredump.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+bucket_name=$S3_BUCKET_NAME
+bucket_prefix=$S3_BUCKET_TEST_PREFIX
+coredump_path=/var/lib/systemd/coredump/
+coredump_pattern=core.mountpoint_s3*
```

Comment:
what is the source for this `mountpoint_s3` suffix? it is not the binary name, right? (as I can see from `cargo test --features=s3_tests --no-run` some of our binaries don't include this suffix, e.g. `target/debug/deps/integration_tests-5661f8ddaa0dd083`, will dumps for those be uploaded?)